app/helpers/application_helper.rb

```diff
@@ -120,7 +120,10 @@ def get_icon_text(icon, text)
   def image_link_to(image_path, url, image_tag_options = {},
                     link_to_options = {})
     link_to url, link_to_options do
-      image_tag image_path, image_tag_options
+      default_alt = File.basename(image_path, ".*").
```

Comment:
Rails appears to have accidentally removed the default alt in v5.2.1 (I think they intended only to deprecate it). The logic here is a straight copy from what Rails used to do, and future-proofs us.